### DISCUSSION TITLE
Osu!lazer only opens when run as adminstrator

### DISCUSSION THREAD

**Discussion by Alkaliii** [ppy/osu#18932] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

I was having an issue installing beatmaps using lazer's built-in browser for a while now. Downloading the osz. files from the internet and trying to open them also didn't work. I decided to ignore the issue hoping it would get fixed in a later update (since every time it failed I was told it was automatically reported to devs). However, several updates have occurred and the issue has not been resolved. Oh well.

A friend of mine suggested I play on stable so I installed stable yesterday and realized a workaround to my beatmap issue by having stable install the beatmap and then importing from stable to lazer. This worked once last night so I planned to try it again this morning. I downloaded some beatmaps and double-clicked on one by mistake while trying to batch select them all and it opened in lazer and installed properly which seemed like a good thing however it wouldn't work when I tried it again. So I closed lazer to see if it would work if the program was closed first, this resulted in lazer no longer opening at all. I then did multiple things to my laser install including reopening the installer I had in my downloads folder over and over to see if I could get a fresh install without destroying my song list and saved scores. Nothing worked, so eventually, I decided to say goodbye to all my progress and uninstall lazer, to see if a reinstall would fix it. I don't remember precisely what occurred but I don't think doing this worked. However, after reinstalling I was able to get lazer to open when I clicked run as administrator.

During all this havoc I believe I corrupted my realm (probably due to my stable import abuse). And some of the few times lazer did run I found all my scores and beatmaps missing. At one point I followed #17213 advice to overwrite my realm with the corrupted one which restored my beatmaps. However I still couldn't run osu as a normal user. This leads me to here I guess. I currently have lazer open as an administrator with my data missing, as well as two other instances of my storage location with different logs (I think). I'm not exactly computer saavy enough to do some of the other diagnostic measures that I have seen here (event viewer, etc) so I'll give up on saving my data and stop tampering with my installation and report my current issue which is the _inability to run the application as a normal user_.

I play on Windows 10 (64 bit)
GPU: AMD Radeon(TM) RX Vega 11 Graphics (Integrated)
CPU: AMD Ryzen 5 2400G
I have no clue what the state of my any of my drivers is.

### Version

2022.205.0-lazer

### Logs

**Current Logs** (Ran as Admin, No Data)
[database.log](https://github.com/ppy/osu/files/9005858/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9005860/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9005861/network.log)
[performance.log](https://github.com/ppy/osu/files/9005862/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9005863/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9005864/updater.log)

**Jank Logs?** (Ran as Admin, Data is intact)
[database.log](https://github.com/ppy/osu/files/9005870/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9005871/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9005873/network.log)
[performance.log](https://github.com/ppy/osu/files/9005874/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9005875/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9005876/updater.log)

**Old Logs** (Wouldn't run as normal, likely due to corrupted realm)
[database.log](https://github.com/ppy/osu/files/9005880/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9005881/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9005882/network.log)
[performance.log](https://github.com/ppy/osu/files/9005883/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9005884/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9005885/updater.log)

**Comment by peppy** [comment, marked as answer]
It won’t run as a normal user because *you ran it as administratir*. Don’t ever run apps as administrator as it means your user won’t have permission to the files that are a created

This is also the reaso. your database is “corrupt” - because the game can’t write to it.

Please start from scratch and never run as administrator.

If you want to keep your data, you will need to change the permissions of all lazer data files to your standard user first. We can’t help with this but you can probably find advice on google.
